client_mock_test.go

```diff
@@ -544,6 +544,106 @@ func TestClientMockClaimHTTP(t *testing.T) {
 	}
 }
 
+func TestClientMockClaimEmail(t *testing.T) {
```

Comment:
Not sure of the hvclient conventions but normally this would be named something like:
TestClient_ClaimEmail

Reply:
No underscores are used in hvclient like that so far, so I was sticking to existing conventions in codebase here.

Reply:
Fine by me.